src/main/java/racing/domain/Car.java

```diff
@@ -2,17 +2,46 @@
 
 public class Car {
     private int location;
+    private String name;
+
+    private static final int NAME_MAX_LENGTH = 5;
 
     public Car(int location) {
         this.location = location;
     }
 
+    public Car(int location, String name) {
+        this.location = location;
+        this.name = nameValidation(name);
+    }
+
+    public String nameValidation(String input) {
+        if (input.length() > NAME_MAX_LENGTH) {
+            throw new IllegalArgumentException("자동차 이름은 5자를 초과할 수 없습니다.");
+        }
+        return input;
+    }
+
     public Car move(int distance) {
         this.location += distance;
-        return new Car(this.location);
+        return new Car(this.location, this.name);
+    }
+
+    public int bigger(int input) {
+        if (input > this.location) return input;
+        return this.location;
+    }
```

Comment:
- brace 를 활용하는게 휴먼에러를 줄일 수 있어요. 그게 아니라면 차라리 삼항연산자로 표현하는게 더 낫다고 생각해요.
- 위 로직은 Math 클래스를 활용하여 아래와 같이 표현할 수도 있겠네요
```suggestion
    public int bigger(int input) {
        return Math.max(input, this.location);
    }
```
- location 값을 가지고 직접 비교하기보다는 Comparator, Comparable 등을 활용하여 객체를 직접 비교하는건 어떨까요